tests/lib/config/config-file.js

```diff
@@ -349,6 +365,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
```

Comment:
You just need to check that the value is unique. Load the same file twice and use `!==` to ensure they're not the same object.